src/main/java/uc/user/logout/LogoutInteractor.java

```diff
@@ -0,0 +1,28 @@
+package uc.user.logout;
+
+import entities.StateTracker;
+
+/**
+ * LogoutInteractor implements logout behaviour by removing currentUser
+ * from the state tracker instance, and changing the view to a login screen.
+ * @layer use cases
+ */
+
+public class LogoutInteractor implements LogoutInputBoundary {
+    private LogoutOutputBoundary presenter;
+    private StateTracker stateTracker;
+
+    public LogoutInteractor(LogoutOutputBoundary presenter, StateTracker stateTracker) {
```

Comment:
We should consider writing Javadoc for the constructor as we do so in multiple other cases throughout our project. Although it may seem redundant to do so, I believe we should stay consistent throughout our project with what we provide documentation for.